### ISSUE TITLE
Swift bool invalid 0xfe

### ISSUE THREAD

**Issue by onmyway133** [onmyway133/notes#278]
let a = true
let b = !a // invalid 0xfe

**Comment by cyupa** [comment]
I have the exact same issue. Have you managed to find why it behaves like this?
The a constant in your case is a computed property in my case so I ended up doing something like:

```
let a = true
let b =  a ? false : true
```

**Comment by onmyway133** [comment]
@cyupa No, I haven't found any answer. It happens constantly on my project, but I can't seem to replicate it in other projects. I'm the Kop fan too ❤️

**Comment by cyupa** [comment]
Yes, I've tried to replicate it in a playground but failed to do so. Probably the memory zone is somehow corrupted, I guess.
You'll never walk alone!
